assets/dev/js/frontend/utils/assets-loader.js

```diff
@@ -42,30 +42,51 @@ export default class AssetsLoader {
 
 			element.onload = () => resolve( true );
 
-			const parent = 'head' === assetData.parent ? assetData.parent : 'body';
-
-			document[ parent ].appendChild( element );
+			this.appendAsset( assetData, element );
 		} );
 	}
+
+	appendAsset( assetData, element ) {
+		const beforeElement = document.querySelector( assetData.before );
+
+		if ( !! beforeElement ) {
+			beforeElement.insertAdjacentElement( 'beforebegin', element );
+			return;
+		}
+
+		const parent = 'head' === assetData.parent ? assetData.parent : 'body';
+
+		document[ parent ].appendChild( element );
+	}
 }
 
-const fileSuffix = elementorFrontendConfig.environmentMode.isScriptDebug ? '' : '.min';
+const isAdmin = document.querySelector( 'body' ).classList.contains( 'wp-admin' );
+
+const assetsUrl = isAdmin
+	? elementorCommonConfig.urls.assets
+	: elementorFrontendConfig.urls.assets;
+
+const fileSuffix = ! isAdmin && elementorFrontendConfig?.environmentMode?.isScriptDebug ? '' : '.min';
 
-const swiperJsSource = elementorFrontendConfig.experimentalFeatures.e_swiper_latest
-	? `${ elementorFrontendConfig.urls.assets }lib/swiper/v8/swiper${ fileSuffix }.js?ver=8.4.5`
-	: `${ elementorFrontendConfig.urls.assets }lib/swiper/swiper${ fileSuffix }.js?ver=5.3.6`;
+const pluginVersion = isAdmin
```

Comment:
I have made this file accessible from the admin side as well, so that we don't have to load all the dialog styling by default on every admin page.